
---
# Result (if docking suceeded or not)
bool success
---
# Feedback 
float32 distance_to_apriltag
std_msgs/String current_state
std_msgs/String current_action_state